charts/argo-cd/values.yaml

```diff
@@ -331,6 +331,49 @@ configs:
       #   ...
       #   -----END CERTIFICATE-----
 
+  # ConfigMap for Config Management Plugins
+  # Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/config-management-plugins/
+  cmp:
+    # -- Create the argocd-cmp-cm configmap
+    create: true
```

Comment:
Let's default to false to retain backwards compatibility and have people enable when needed. It's also aligning with upstream defaults.

Reply:
@mbevc1 I have set it to false.

Reply:
Thanks, could you also update the docs (run the script) to reflect proper default please?